### DISCUSSION TITLE
Which theme/snippet is used to make the callout blocks look like this?

### DISCUSSION THREAD

**Discussion by benjiboi** [efemkay/obsidian-modular-css-layout#48] [Q&A] [answered]
Absolutely love the look of the callout blocks in the example. But can't seem to find anything about which snippet or theme is used
![hero-mc-callout](https://github.com/efemkay/obsidian-modular-css-layout/assets/20013920/a4e879a3-34ad-4fd1-9d83-5b42c952e95e)
Thought I'd ask here first since the pic is in the ReadMe file.

Anyway, big thanks for the multicolumn css

**Comment by efemkay** [comment, marked as answer]
hey there. yeah previously i would make a footnote which theme i use for screenshot (then i become lazy 😂). anyway, that's Minimal theme with callout type "Outlined"

![image](https://github.com/efemkay/obsidian-modular-css-layout/assets/42369515/664f7492-1196-4683-b5d7-cae99f326aa5)

**Reply by chrjohns** [reply]
How about the setting for the Prism theme you use here 
![](https://raw.githubusercontent.com/efemkay/obsidian-modular-css-layout/main/docs/assets/showcases/showcase-mc-callout-and-list.png)

What are the color settings for the callouts?  They look great!

**Reply by efemkay** [reply]
for that showcase, i only use the default settings of Prism's light theme.
![image](https://github.com/efemkay/obsidian-modular-css-layout/assets/42369515/3c1263e5-0964-4919-83cc-a3e7d1a4cd24)
